e2e/README.md

```diff
@@ -1,21 +1,18 @@
-# All-of-Us Test Automation with Puppeteer
-Test using Jest as test runner and Puppeteer to perform automation tests in local installed Chrome browser.
+# All-of-Us Test Automation with Puppeteer in Chrome
 
-### Try running tests in localhost
-#### clone github project
+#### Clone github project
 - `git clone git@github.com:all-of-us/workbench.git`
 
-#### set up login credential
+#### Set up login credential
 - copy `.env.sample` file. Save new file as `.env` (do not commit `.env` file to version control)
 - edit `.env` to provide workbench login credential
```

Comment:
Add text here describing how it needs to be an existing user in the Test environment without 2FA

Reply:
done